cluster/manifests/z-karpenter/deployment.yaml

```diff
@@ -0,0 +1,93 @@
+{{ if eq .Cluster.ConfigItems.karpenter_pools_enabled "true"}}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: karpenter
+  namespace: kube-system
+  labels:
+    application: kubernetes
+    component: karpenter
+spec:
+  replicas: 2
+  strategy:
+    rollingUpdate:
+      maxUnavailable: 1
+  selector:
+    matchLabels:
+      deployment: karpenter
+  template:
+    metadata:
+      labels:
+        deployment: karpenter
+        application: kubernetes
+        component: karpenter
+      annotations:
+        logging/destination: "{{.Cluster.ConfigItems.log_destination_infra}}"
+    spec:
+      dnsConfig:
+        options:
+        - name: ndots
+          value: "1"
+      dnsPolicy: Default
+      serviceAccountName: karpenter
+      securityContext:
+        fsGroup: 1000
+      priorityClassName: "{{ .Cluster.ConfigItems.system_priority_class }}"
+      tolerations:
+      - key: node.kubernetes.io/role
+        value: master
+        effect: NoSchedule
+      containers:
+        - name: controller
+          image: "container-registry.zalando.net/teapot/karpenter-controller:v0.27.3-main-9"
+          env:
+            - name: KUBERNETES_MIN_VERSION
+              value: "1.19.0-0"
```

Comment:
What is the significance of this? Does it make sense to change it to 1.22/23?

Reply:
checking...

Reply:
![Screenshot 2023-04-27 at 09 51 23](https://user-images.githubusercontent.com/1551247/234796110-bf09b4cf-5e91-4479-8617-23b307ee2d75.png)
then I will change it to 1.22